source/getting_started.rst

```diff
@@ -4,6 +4,8 @@ Getting Started
 ===============
 This section provides instructions on how to download and install the WEC-Sim code.
 
+.. note::
+    download and install and test
 
```

Comment:
added test

Reply:
added a [test install ](http://wec-sim.github.io/WEC-Sim/getting_started.html#step-3-test-the-installation)section, and a [software tests ](http://wec-sim.github.io/WEC-Sim/overview.html#software-tests)section.